server/views/partials/layout.njk

```diff
@@ -63,4 +64,44 @@
   <script src="/assets/govuk/all.js"></script>
   <script src="/assets/govukFrontendInit.js"></script>
   <script src="/assets/moj/all.js"></script>
+
+    {% if env and env == 'PRE-PRODUCTION' %}
+  <script src="https://cdn.lr-in.com/LogRocket.min.js" crossorigin="anonymous"></script>
```

Comment:
This works just for dev? even though its called 'PRE-PRODUCTION'?

Reply:
updated PR title, should be preprod only